compiler/core/static/swift/AttributedFont.swift

```diff
@@ -17,8 +17,7 @@ public class AttributedFont {
     size: CGFloat = UIFont.systemFontSize,
```

Comment:
I think the changes in this file (and `examples/generated/test/swift/AttributedFont.swift`) are fine. This file is essentially a source file, since it's copied over into the generated code directory automatically. So I think we _should_ continue to lint the `compiler` directory for now. :+1: